.gitmodules

```diff
@@ -27,7 +27,6 @@
 [submodule "vendor/cmsis_5"]
 	path = vendor/cmsis_5
 	url = https://github.com/ARM-software/CMSIS_5.git
-
```

Comment:
please drop this difference, it's just noise

Reply:
> please drop this difference, it's just noise

I'm not sure if I understand. This is not directly related to this pull request but what is wrong about removing an extra empty line in `.gitmodules`, @matejcik ?

Reply:
i haven't looked at the file closely and assumed that there are empty lines between every entry. after your comment i see that it's the opposite. in which case sure, feel free to remove it